tests/test_redeeming.py

```diff
@@ -61,105 +60,92 @@ async def channel_redeem_tickets(api: HoprdAPI, channel: str):
-        self, src: str, dest: str, swarm7: dict[str, Node]
-    ):
-        ticket_price = await get_ticket_price(swarm7[src])
+    @pytest.mark.parametrize("route", make_routes([1], barebone_nodes()))
+    async def test_hoprd_should_create_redeemable_tickets_on_routing_in_1_hop(self, route, swarm7: dict[str, Node]):
+        ticket_price = await get_ticket_price(swarm7[route[0]])
         # send 90% of messages before ticket aggregation would kick in
         message_count = int(TICKET_AGGREGATION_THRESHOLD / 10 * 9)
-        assert message_count < TICKET_AGGREGATION_THRESHOLD
 
         async with create_bidirectional_channels_for_route(
-            [swarm7[src], swarm7[dest]], message_count * ticket_price, ticket_price
+            [swarm7[hop] for hop in route], message_count * ticket_price, ticket_price
         ) as channels:
-            statistics_before = await swarm7[dest].api.get_tickets_statistics()
+            relay = swarm7[random.choice(route[1:-1])]
+            statistics_before = await relay.api.get_tickets_statistics()
 
-            await session_send_and_receive_packets_over_single_route(
+            await basic_send_and_receive_packets_over_single_route(
                 message_count,
-                [swarm7[src], swarm7[dest]],
+                [swarm7[hop] for hop in route],
             )
 
             await asyncio.wait_for(
                 check_unredeemed_tickets_value(
-                    swarm7[dest], statistics_before.unredeemed_value + message_count * ticket_price
+                    relay, statistics_before.unredeemed_value + message_count * ticket_price
                 ),
                 30.0,
             )
 
             # ensure ticket stats are updated after messages are sent
-            statistics_after = await swarm7[dest].api.get_tickets_statistics()
+            statistics_after = await relay.api.get_tickets_statistics()
 
             assert statistics_after.redeemed_value == statistics_before.redeemed_value
 
-            assert await swarm7[dest].api.channel_redeem_tickets(channels.fwd_channels[0].id)
+            assert await relay.api.channel_redeem_tickets(channels.fwd_channels[0].id)
 
-            await asyncio.wait_for(check_all_tickets_redeemed(swarm7[dest]), 120.0)
+            await asyncio.wait_for(check_all_tickets_redeemed(relay), 120.0)
 
             # ensure ticket stats are updated after redemption
-            statistics_after_redemption = await swarm7[dest].api.get_tickets_statistics()
+            statistics_after_redemption = await relay.api.get_tickets_statistics()
             assert (statistics_after_redemption.redeemed_value - statistics_after.redeemed_value) == (
                 message_count * ticket_price
             )
             assert statistics_after_redemption.unredeemed_value == 0
 
     @pytest.mark.asyncio
-    @pytest.mark.skip(reason="ticket aggregation is not implemented as a session protocol yet")
```

Comment:
Are you sure? Ticket aggregation in 3.0 is disabled.

Reply:
You disabled the test incorrectly - it can run without aggregation, with just a minimal change.